src/main/java/com/truelayer/java/paymentsproviders/entities/searchproviders/SearchPaymentProvidersRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.truelayer.java.paymentsproviders.entities.searchproviders;
+
+import com.truelayer.java.entities.CurrencyCode;
+import com.truelayer.java.payments.entities.CountryCode;
+import com.truelayer.java.payments.entities.CustomerSegment;
+import com.truelayer.java.payments.entities.ReleaseChannel;
+import java.util.List;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Builder
+@Getter
+@ToString
+@EqualsAndHashCode
+public class SearchPaymentProvidersRequest {
+    private List<CountryCode> countries;
+
+    private List<CurrencyCode> currencies;
+
+    private ReleaseChannel releaseChannel;
+
+    private List<CustomerSegment> customerSegments;
+
+    private Capabilities capabilities;
```

Comment:
I could not reuse the same `Capabilities` object already defined in the `entities` folder (currently used for responses) because in the request `BankTransferCapabilites`, `VrpSweepingCapabilities` and `VrpCommercialCapabilities` are empty objects (whereas they have additional properties in the response).

I initially considered reusing the same objects anyway as adding additional properties in the request was not creating side effects in the APIs, but then I thought it might have led to confusion clients if they were expecting to use those additional properties for filtering out providers